examples/nodejs/storage-load-gen/README.ja.md

```diff
@@ -0,0 +1,95 @@
+# Node.js クライアント SDK
```

Comment:
I think maybe we should remove this and let the translators do it if it is needed?